src/content/self-hosted/administration/configuration.mdx

```diff
@@ -1299,12 +1300,14 @@ Disable if you want to bring your own certificates and/or certificate authority.
 - `privateCA.enabled`: Set to true when using a private certificate authority.
 - `privateCA.secret.name`: The name of the secret that stores the private certificate authority's certificate.
 - `privateCA.secret.key`: The key in the secret that stores the private certificate authority's certificate.
+- `annotations`: Annotations to add to the wildcard certificate generated by Okteto.
 
 ```yaml
 wildcardCertificate:
   create: true
   duration: 2160h0m0s
   name: default-ssl-certificate
+  annotations: {}
```

Comment:
would it make more sense to have a working example of the annotations rather than the empty value?

Reply:
Added an example of annotation